substrate/frame/identity/src/lib.rs

```diff
@@ -150,6 +150,32 @@ pub mod pallet {
 	use super::*;
 	use frame_support::pallet_prelude::*;
 
+	#[cfg(feature = "runtime-benchmarks")]
+	pub trait BenchmarkHelper<Public, AccountId, Signature> {
+		fn signer() -> (Public, AccountId);
+		fn sign(signer: &Public, message: &[u8]) -> Signature;
```

Comment:
```suggestion
		/// Sign the given `message` with a random account.
		///
		/// Should return the public key of this random account and the signature.
		fn sign_message(message: &[u8]) -> (Public, Signature);
```

Just make this one function. In the benchmarking code you can get the `AccountId` by calling `into_account` on the `public_key`.

Reply:
I get the intention of reducing the interface to only one function, but still I don't like the idea, mainly because:

1) Again it differs from `pallet-nfts` [equivalent code](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/nfts/src/lib.rs#L92-L93).
2) This prevents from signing twice with the exact same account. Not that it's needed now but adds unnecessary constrains that might bite devs in future iterations of the pallet and likely force to revert the changes and introduce extra noise.

Just my two cents. If you still prefer your proposed approach I'll happily change it, but I thought it was worth discussing.

Reply:
Done in [00e81d4](https://github.com/paritytech/polkadot-sdk/pull/8179/commits/00e81d4eac066d339fe79e6128f3e9381eab20db).